src/Menu/Menu.js

```diff
@@ -242,6 +248,7 @@ class Menu extends React.Component {
     this.setState({
       focusIndex: nextProps.disableAutoFocus ? -1 : selectedIndex >= 0 ? selectedIndex : 0,
       keyWidth: nextProps.desktop ? 64 : 56,
+      isKeyboardFocused: nextProps.keyboardFocused || this.state.isKeyboardFocused,
```

Comment:
Good question :smile: 
Do you think we could use only `nextProps.keyboardFocused` here?